packages/venia-ui/lib/components/LegacyMiniCart/productOptions.js

```diff
@@ -15,11 +15,15 @@ const ProductOptions = props => {
                 const key = `${option_label}${value_label}`;
 
                 return (
-                    <Fragment key={key}>
-                        <dt className={classes.optionLabel}>
-                            {option_label} : {value_label}
-                        </dt>
-                    </Fragment>
+                    <div key={key} className={classes.optionLabel}>
+                        {/**
+                         * Added `\u00a0` to simulate a space character.
+                         * We can't use ` ` because JS string lirerals logic
+                         * does not allow leading or trailing space chaacters.
+                         */}
+                        <dt>{`${option_label} :\u00a0`}</dt>
```

Comment:
Not sure if this is the right way to handle it, open for suggestions.

Reply:
Consider using CSS (`margin`, `padding`, `gap`) on the `dd` element?

Reply:
I did, but the gap has to be exactly `1 space` to match the UX. I even tried `<dt>{option_label + ' : '}</dt>`, it did not work. The trailing space was removed.

Reply:
> I did, but the gap has to be exactly `1 space` to match the UX. I was not able to nail down the exact number of a space character.

We don't need to match designs to the pixel. Just approximate it. 😅 

> Any suggestions on what would be the size of a space character?

For Venia in general, I work in increments of `0.25rem` when estimating spacing. That should work fine here—make sure to add classnames to the `dt` and `dd` elements.

```
// structure
div.optionLabel
  dt.optionName
  dd.optionValue

// css
.optionValue
  margin-left: 0.25rem
```

Reply:
> We don't need to match designs to the pixel. Just approximate it. 😅

![image](https://user-images.githubusercontent.com/35203638/88841332-702b9400-d1a3-11ea-82f0-c0ca9ca86261.png)

I had to in this case because there are other options that don't fall into `ProductOptions` like quantity and if we can't nail the space correctly, they will look misaligned. 

That said, nice suggestion about `0.25rem`, that exactly 4px and works like a gem 👍 .